man/rustc.1

```diff
@@ -242,6 +242,52 @@ full debug info with variable and type information.
 \fBopt\-level\fR=\fIVAL\fR
 Optimize with possible levels 0\[en]3
 
+.SH ENVIRONMENT VARIABLES
+
+.TP
+\fBRUST_TEST_TASKS\fR
+The test framework Rust provides executes tests in parallel. This variable sets
+the maximum number of threads used for this purpose.
+
+.TP
+\fBRUST_TEST_NOCAPTURE\fR
+A synonym for the --nocapture flag.
+
+.TP
+\fBRUST_REGION_GRAPH_NODE\fR
+Works with `-Z print-region-graph`. Only produces output for the node ID provided
+via this variable.
+
+.TP
+\fBRUST_REGION_GRAPH\fR
+Works with `-Z print-region-graph`. Rather than being output to `/tmp`, will
+instead output to this path. Portions of the path containing `%` will be replaced
+with the node's ID.
+
+.TP
+\fBRUST_TARGET_PATH\fR
+Used with `--target=TRIPLE` to provide a search path for TRIPLE.json.
+
+.TP
+\fBRUST_MIN_STACK\fR
+Sets the minimum stack size for new threads.
+
+.TP
+\fBRUST_BACKTRACE\fR
+If set, produces a backtrace in the output of a program which panics.
+
+.TP
+\fBDXR_RUST_TEMP_FOLDER\fR
+A temporary folder for the analysis output of trans.
+
+.TP
+\fBRUST_THREADS\fR
+The default number of threads.
```

Comment:
Ah yeah it's just in a function that's no longer used except in libtest which probably shouldn't be using it anyway!